sunpy/map/mapbase.py

```diff
@@ -1526,7 +1526,7 @@ def _pv_values(self):
         """
         Return any PV values in the metadata.
         """
-        pattern = re.compile('pv[0-9]_[0-9]{1,2}$', re.IGNORECASE)
+        pattern = re.compile(r'pv[1-9]\d?_(?:0|[1-9]\d?)$', re.IGNORECASE)
```

Comment:
Ok, why `0|[1-9]` rather than `\d`?

Wouldn't `\d\d?` have the same effect?

Reply:
The second option of the or is `[1-9]\d?`, not just `[1-9]`.  That is, match either 0 or a up-to-two-digit number that doesn't start with a zero.

Reply:
Oh man regex is *fun*.